site/docs/docs.md

```diff
@@ -39,3 +43,7 @@ In Vega-Lite, a specification can have the following top-level properties.
 | [mark](mark.html)    | String        | The mark type.  One of `"bar"`, `"circle"`, `"square"`, `"tick"`, `"line"`, `"area"`, `"point"`, and `"text"` (text table). |
 | [encoding](encoding.html) | Object   | A key-value mapping between encoding channels and definition of fields. |
 | [config](config.html)   | Object     | Configuration object. |
+
+## [](#lib) The Vega-Lite Library
```

Comment:
Change if you like. Right now this only has one think defined. We need to overhaul this page in general but that's beyond the scope of this PR.